content/en/introduction.org

```diff
@@ -10,12 +10,12 @@
 
 Hewwo hewwo nyaoo nyaoo~~
 
-My name is Yayoi Shizumi (静海やよい in Japanese) and I am a magical fairy cat princess from a mystical realm near the North West Passage! I was sent to this realm to protect it. Unfortunately, my travel here was far from uneventful and I am no longer able to speak. I have had to find new ways to communicate in order to protect those I was sent to protect, my kittens!
+My name is Yayoi Shizumi (静海やよい in Japanese) and I am a magical fairy cat princess from a mystical realm near the North West Passage! I was sent to this realm to protect it. Unfortunately, my travel here was far from uneventful and I am no longer able to speak. I have had to find new ways to communicate in order to protect those I was sent +to protect+ (here for, "protect" seems redundant here), my kittens!
```

Comment:
I think you're right, "protect" does seem redundant. "I was sent here for" doesn't quite seem like something i'd say though. How about "I have had to find new ways to communicate in order to fulfill my mission (or task)"?

Reply:
I like "I have had to find new ways to communicate in order to fulfill my mission", but you would lose the reference about your kittens, so maybe something like:

```suggestion
My name is Yayoi Shizumi (静海やよい in Japanese) and I am a magical fairy cat princess from a mystical realm near the North West Passage! I was sent to this realm to protect it. Unfortunately, my travel here was far from uneventful and I am no longer able to speak. I have had to find new ways to communicate in order to fulfill my mission, and protect my kittens!
```